src/panels/lovelace/components/hui-badge-edit-mode.ts

```diff
@@ -231,8 +231,8 @@ export class HuiBadgeEditMode extends LitElement {
     fireEvent(this, "ll-edit-badge", { path: this.path! });
   }
 
-  private _deleteBadge(): void {
-    fireEvent(this, "ll-delete-badge", { path: this.path! });
+  private _deleteBadge({ silent }: { silent: boolean }): void {
```

Comment:
Why not using here a boolean too, like in DeleteBadgeParams?

Reply:
I simplified the implementation in the last commit. 👍